types/docopt/index.d.ts

```diff
@@ -6,9 +6,7 @@
 /**
  * @param doc should be a string with the help message, written according to rules of the docopt language.
  */
-declare function docopt(doc: string, options: DocoptOption): any;
-
-export = docopt;
+export function docopt(doc: string, options: DocoptOption): any;
 
 interface DocoptOption {
```

Comment:
If you would like consumers of this declaration file to be able to reference DocoptOption, you should export it as well.

Reply:
Good call! It was not the case in the original type and was not the scope of this PR so I didn't change that initially, but it's a good idea so I added a commit to do just that.